### PR TITLE
add snitin315 as maintainer of appcenter plugin

### PR DESCRIPTION
**Pull request by snitin315**
# Description

I would like to adopt the app center plugin.

https://plugins.jenkins.io/appcenter/
https://github.com/jenkinsci/appcenter-plugin/

Thread - https://groups.google.com/g/jenkinsci-dev/c/vpPYihV87NM/m/TfmSF6olAQAJ

/cc @mezpahlan 

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by genesisthomas** [comment]
Dear @snitin315, my PR tests are also with the similar error: "User name not known to Artifactory: genesisthomas" 
Kindly let me know if you found a way to fix it.

**Comment by snitin315** [comment]
Maybe it's because I'm not a member of the organization on GitHub.

**Comment by genesisthomas** [comment]
> Maybe it's because I'm not a member of the organization on GitHub.

Thanks for your response. In my case, I have admin access to the plugins project, I have even pushed code to it but still Artifactory doesn't know me :)

**Comment by mezpahlan** [comment]
Hi chaps have you read https://www.jenkins.io/doc/developer/publishing/releasing/ and https://www.jenkins.io/doc/developer/publishing/artifact-repository/?

If I remember you also need a user in Artifactory. Try logging in there and seeing if you are set up as a user. Sorry it's been a while since I did this so maybe wait for a Jenkins Core team member to help further.

File: permissions/plugin-appcenter.yml [hunk 1]
```diff
@@ -5,3 +5,4 @@ paths:
 - "io/jenkins/plugins/appcenter"
 developers:
 - "mezpahlan"
+- "snitin315"
```
Comment (snitin315) [inline permissions/plugin-appcenter.yml:8]:
```suggestion
- "nitin_kumar"
```

**Comment by mezpahlan** [comment]
Hi I'm the existing maintainer of the appcenter plugin. 

Approve and also please give merge permissions to @snitin315 .

**Comment by timja** [comment]
@snitin315 please login to https://repo.jenkins-ci.org/webapp/#/home using username `nitin_kumar`

**Pull request merged**